src/Controls/src/Core/Platform/Windows/CollectionView/ItemContentControl.cs

```diff
@@ -266,5 +269,14 @@ protected override WSize MeasureOverride(WSize availableSize)
 		{
 			return double.IsInfinity(value) ? 0 : value;
 		}
+
+		bool CanMeasureContent(FrameworkElement frameworkElement)
+		{
+			// Measure the SwipeControl before has loaded causes a crash on the first layout pass
+			if (frameworkElement is SwipeControl swipeControl && !swipeControl.IsLoaded)
```

Comment:
Is this exclusive to SwipeControl? Should this just check for a `Control` that `IsLoaded` ?

Reply:
A far as I reviewed cannot Measure the SwipeControl with left or right items before is loaded, but, not happens in the rest of the controls.